ranger/gui/ui.py

```diff
@@ -116,14 +116,17 @@ def initialize(self):
             if self.settings.update_tmux_title and 'TMUX' in os.environ:
                 try:
                     self._tmux_automatic_rename = check_output(
-                        ['tmux', 'show-window-options', '-v', 'automatic-rename']).strip()
+                        ['tmux', 'show-window-option', '-v', 'automatic-rename']).strip()
+                    if self._tmux_automatic_rename == 'off':
+                        self._tmux_automatic_rename = 'on'
                 except CalledProcessError:
                     self._tmux_automatic_rename = None
 
         self.update_size()
         self.is_on = True
 
         if self.settings.update_tmux_title and 'TMUX' in os.environ:
+            check_output(['tmux', 'rename-window', 'Ranger'])
             sys.stdout.write("\033kranger\033\\")
```

Comment:
Turns out this was broken in tmux 2.7, cf. tmux/tmux#1322.
This is pretty sad because I think it's cross compatible with screen. Let's try to stay screen compatible.

Reply:
I guess setting `allow-rename` first would have the control sequence work again. But that's dirty. We should just tell the user in `rc.conf` that they have to allow renaming and then emit a warning if `update_tmux_title` is set but `allow-rename` is not.